packages/compiled-assets/README.md

```diff
@@ -20,13 +20,22 @@ Create a directory of assets that you wish to bundle, e.g. `assets/`. Within tha
 You can locate shared code in directories inside this directory. As long as those files aren't in the root of the `scripts/` directory, they won't become separate bundles.
 
 ```text
-├── assets/
-│   ├── scripts/
-|   │   ├── lib/
-|   │   │   ├── shared-code.ts
-|   │   │   └── more-shared-code.ts
-|   │   ├── foo.ts
-│   |   └── bar.ts
+assets
+└── scripts
+    ├── bar.ts
+    ├── foo.ts
+    └── lib
+        ├── more-shared-code.ts
+        └── shared-code.ts
+```
+
+All of these assets will be processed as a CJS file for compatibility reasons. You can place additional assets in in the `assets/scripts/esm-bundles/` directory, which will be processed as ESM files with code splitting. This is useful for libraries that can be loaded asynchronously, like Preact components.
```

Comment:
Points of clarification:

- The files themselves can still use ESM syntax.
- The actual output format is an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE), as CommonJS doesn't run natively in the browser.

Reply:
Changed in d1b50953c815db929092da183ff68b34e7dfd6b5. Let me know if the updated copy looks good.